### ISSUE TITLE
Are the docs on the site for 0.14 or 1.0 alpha?

### ISSUE THREAD

**Issue by rchrdnsh** [vuejs/vuepress#911]
Going through the docs on the Vuepress site for 0.14 or 1.0? Nothing seems to really be working very well at this point 🤔

**Comment by rchrdnsh** [comment]
Yeah, nothing seems to work right and i’m getting error messages now that break the dev server. Will try again with @next and see if the same stuff is happening.

**Comment by ulivz** [comment]
So why the documentation website works well?

**Comment by ulivz** [comment]
Please do not pollute issue.

**Comment by rchrdnsh** [comment]
apologies, but I don’t understand what ‘pollute the issue’ means.

**Comment by rchrdnsh** [comment]
Here is a screenshot of the error I'm getting when I...

```
yarn global add vuepress@next
``` 
and then...
```
echo '# Hello VuePress' > README.md
```
and then run...
```
vuepress dev
```

<img width="1440" alt="screenshot 2018-10-06 23 56 43" src="https://user-images.githubusercontent.com/7884263/46579318-b6f33d00-c9c3-11e8-9409-19b5b033574d.png">

I can't seem to get beyond even the first three steps to make a vuepress app, so I'm not sure what to do next.

**Comment by mAKEkr** [comment]
can you tried on local install? maybe vuepress global install has problem on some environment.

**Comment by rchrdnsh** [comment]
apologies, but I'm not sure how to make this work. I installed `vuepress@next` locally, I think? then I added your dev line to the package.json file, like this...
```
{
  "dependencies": {
    "vuepress": "^1.0.0-alpha.8"
  },
  "dev": "vuepress dev {docs}"
}
```
and I run `yarn run dev` and I get this message...

<img width="828" alt="screen shot 2018-10-07 at 10 41 59 am" src="https://user-images.githubusercontent.com/7884263/46584888-d74fe580-ca1d-11e8-806b-2e64462cc1c3.png">

so, I'm not sure what to do next...

**Comment by mAKEkr** [comment]
I'm sorry. my explanation was immatured.
fix your `packages.json` file of these context.
```js
"dev": "vuepress dev {docs}"
```
to

```js
"scripts": {
  "dev": "vuepress dev docs"
}
```

and try `yarn run dev` command again.

**Comment by rchrdnsh** [comment]
ok, so a bit of progress. It seems to run ok now, but I'm taken to a 404 page, so not sure what to do next...

**Comment by jrappen** [comment]
This seems to be a duplicate of #908 which was fixed in vuejs/vuepress@af80c4b from [`v1.0.0-alpha.17`](https://github.com/vuejs/vuepress/releases/tag/v1.0.0-alpha.17) according to @ulivz

**Comment by rchrdnsh** [comment]
ok, I'll close this for now then and check out the latest build.

**Comment by rchrdnsh** [comment]
ok, great, will check out the latest build then :-)
